requirements.txt

```diff
@@ -0,0 +1,254 @@
+#
+# This file is autogenerated by pip-compile with python 3.9
+# To update, run:
+#
+#    pip-compile
```

Comment:
this is a superset of what is required for this project -- may be worth pruning it in the future

Reply:
What is superfluous? The direct requirements, in requirements.in, are I think the minimal ones? I think without each of the four listed there I saw failures?